geometry_msgs/PoseStamped palm_approach_pose_world
geometry_msgs/PoseStamped palm_goal_pose_world
---
bool success
float64 fraction
trajectory_msgs/JointTrajectory trajectory
